src/content/docs/infrastructure/host-integrations/host-integrations-list/haproxy-monitoring-integration.mdx

```diff
@@ -10,52 +10,47 @@ redirects:
   - /docs/haproxy-integration-new-relic-infrastructure
 ---
 
-Our HAProxy [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) collects and sends inventory and metrics from your HAProxy instance to the New Relic platform, where you can aggregate and visualize key performance metrics for the HAProxy server, frontends, and backends.
+Our HAProxy [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) collects and sends inventory and metrics from your HAProxy instance to the New Relic platform, where you can aggregate and visualize key performance metrics for the HAProxy server, frontends, and backends.   
 
 Read on to install the integration, and to see what data we collect.
 
 ## Compatibility and requirements [#comp-req]
 
-Our integration is compatible with HAProxy versions 1.3 - 2.x.
-
-Before installing the integration, make sure that you meet the following requirements:
+Before installation, ensure you meet these requirements:
 
+* A New Relic account. Don't have one? [Sign up for free!](https://newrelic.com/signup) No credit card required.
+* HAProxy versions 1.3 - 2.x.
 * HAProxy statistics page is enabled and accessible.
-* If HAProxy is **not** running on Kubernetes or Amazon ECS, you must [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic) on a host running HAProxy. Otherwise:
-  * If running on Kubernetes, see [these requirements](/docs/monitor-service-running-kubernetes#requirements).
-  * If running on ECS, see [these requirements](/docs/integrations/host-integrations/host-integrations-list/monitor-services-running-amazon-ecs).
 
 ## Quick start [#quick]
 
-Instrument your HAProxy instance quickly and send your telemetry data with guided install. Our guided install creates a customized CLI command for your environment that downloads and installs the New Relic CLI and the infrastructure agent.
+If you're not running HAProxy in Kubernetes or ECS environments, we recommend our guided install. Our guided install uses our infrastructure agent and our CLI to set up the HAProxy integration, and discovers other applications and log sources running in your environment and then recommends which ones you should instrument.
```

Comment:
This might actually have a different process... AFAIK K8s itself is an integration, so the person might need two integrations: Haproxy and K8s... I'd x2 check this with your SME. 

also good area to update whatever direction you go with with the CLI copy

Reply:
@paologallinaharbur Would you mind taking a look at this and letting me know if I need to change the copy? Thanks!

Reply:
In general, instrumenting an integration on ECS or on k8s is something that could be quite difficult.

As far as I know, the guided install helps install a monitoring solution for the ECS and K8s themself OR the integration running on Windows.
 
However, to monitor the service running on those environments you are basically on your own and you need to refer to the docs.

Therefore I think the text is correct "If you're NOT running". However, reading it quickly I missed the NOT

Reply:
Moreover, the guided install is for Linux only:
<img width="619" alt="Screenshot 2023-09-25 at 17 23 40" src="https://github.com/newrelic/docs-website/assets/43335750/74674a2d-d324-4437-82d9-6c4ca9c3623f">

Reply:
Thanks @paologallinaharbur! I appreciate you!